### ISSUE TITLE
Wifi network bridge without NAT using systemd-networkd

### ISSUE THREAD

**Issue by rdmark** [akuker/RASCSI#917]
Discord user piranger found a way to make a wiki network bridge work without NAT using systemd-networkd: https://grkidwell.github.io/rascsiwifi/

This ticket is to recreate their steps and look into making this the default/recommended configuration.

**Comment by rdmark** [comment]
Some thoughts on this after reading through piranger's solution:
- avahi-daemon is disabled, which is something that is used by CUPS, Netatalk and other networking applications that often run side by side with RaSCSI. Reading up on this, it turns out that systemd-resolved, which is part of systemd-networkd, is a conflicting implementation of service discovery. According to the Arch wiki there are workarounds that may be sufficient. This will require testing. https://wiki.archlinux.org/title/Avahi#systemd-resolved_prevents_nss-mdns_from_working
- The wifi router you use must be configured so that its DHCP ranges leave some upper address space for the proxy ARP subnetting. For one, this puts a spanner in the works for having an automated setup script, since we cannot anticipate everyone's wifi router capabilities and settings. And additionally, the particular wifi router that I have does not provide that kind of fine grained control over DHCP. I will have to acquire a secondary wifi router to proceed with the experiment.

In summary, this does not seem like a feasible general-purpose approach for RaSCSI. Someone who is more knowledgeable with systemd-networkd might chime in and provide guidance, however! :)
